pkg/controller/cainjector/sources.go

```diff
@@ -149,20 +149,20 @@ func (c *certificateDataSource) ApplyTo(ctx context.Context, mgr ctrl.Manager, s
 	}
 
 	if err := controller.Watch(source.NewKindWithCache(&cmapi.Certificate{}, ca),
-		&handler.EnqueueRequestsFromMapFunc{ToRequests: &certMapper{
```

Comment:
Note: this and below changes are also related to changes in controller-runtime `v0.7.0`, see https://github.com/kubernetes-sigs/controller-runtime/pull/1207.